sr_description/mujoco_models/shared_ur10_options.xml

```diff
@@ -0,0 +1,10 @@
+<mujoco>
+    <actuator>
+        <motor name="ra_shoulder_pan_joint" joint="ra_shoulder_pan_joint" ctrllimited="false" ctrlrange="-6.28319 6.28319"/>
+        <motor name="ra_shoulder_lift_joint" joint="ra_shoulder_lift_joint"  ctrllimited="false" ctrlrange="-6.28319 6.28319"/>
+        <motor name="ra_elbow_joint" joint="ra_elbow_joint" ctrllimited="false" ctrlrange="-3.14159 3.14159"/>
+        <motor name="ra_wrist_1_joint" joint="ra_wrist_1_joint" ctrllimited="false" ctrlrange="-6.28319 6.28319"/>
+        <motor name="ra_wrist_2_joint" joint="ra_wrist_2_joint" ctrllimited="false" ctrlrange="-6.28319 6.28319"/>
+        <motor name="ra_wrist_3_joint" joint="ra_wrist_3_joint" ctrllimited="false" ctrlrange="-6.28319 6.28319"/>
+    </actuator>
```

Comment:
Maybe UR assets (mesh definitions) could go here?

Reply:
i used a new assets file like we do for hand

Reply:
They're in shared_assers.xml, which means they're loaded for every robot, including ones without arms. I'm suggesting making an e.g. ur_shared_assets.xml, and including that file here.

Reply:
ah yes I mean in my new change which was not pushed yet :)